Source/lighting.cpp

```diff
@@ -919,10 +919,9 @@ void MakeLightTable()
 		for (int i = 0; i < 8; i++) {
 			for (int k = 0; k < 16; k++) {
 				for (int l = 0; l < 16; l++) {
-					double fs = (BYTE)sqrt((double)(8 * l - j) * (8 * l - j) + (8 * k - i) * (8 * k - i));
-					fs += fs < 0 ? -0.5 : 0.5;

```

Comment:
This did absolutely nothing useful. BYTE is an unsigned integral type so fs is always a positive whole number. It then gets immediately converted back to an integral type which rounds toward zero, dropping the .5 value.